src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/filters/FilterRequest.kt

```diff
@@ -58,6 +63,10 @@ data class GrpcFilter(val filterName: String, val filter: Filter) : FilterReques
         return filter.conjunct
     }
 
+    override fun isStrict(): Boolean {
+        TODO("Not yet implemented")

```

Comment:
Please, use default **false** value, otherwise wa can't create filter from grpc because it method always call in all filters